prefs.ini

```diff
@@ -29,18 +29,33 @@ plugins.flashBlock.enabled = true
 ################################
 
 [stage]
-browser.safebrowsing.provider.mozilla.updateURL = https://shavar.stage.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.updateURL = https://shavar.stage.mozaws.net/list?client=foo&appver=1&pver=2.2
+
+browser.safebrowsing.provider.mozilla.downloads = https://shavar.stage.mozaws.net/downloads?client=foo&appver=1&pver=2.2
+
+base_domain = tracking-protection.stage.mozaws.net
+
+#https://shavar.stage.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
 
 [pre-prod]
-browser.safebrowsing.provider.mozilla.updateURL = https://shavar.prod.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.updateURL = https://shavarprodtest.prod.mozaws.net/list?client=foo&appver=1&pver=2.2%27
+
+base_domain = shavarprodtest.prod.mozaws.net
 
 [prod]
-browser.safebrowsing.provider.mozilla.updateURL = https://shavar.services.mozilla.com/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.updateURL = https://shavar.services.mozilla.com/list?client=foo&appver=1&pver=2.2
+
+browser.safebrowsing.provider.mozilla.downloads = https://shavar.services.mozilla.com/downloads?client=foo&appver=1&pver=2.2
+
+base_domain = tracking-protection.cdn.mozilla.net
+
+#https://shavar.services.mozilla.com/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
```

Comment:
when we hack today, let's figure out if one of the other tests actually requires these extra params written in this way or if we can drop this altogether.  I'm curious what those values are....